templates/tripleo_myservice/defaults/main.yml.j2

```diff
@@ -0,0 +1,28 @@
+{%- import 'templates/macros.yml' as macros -%}
```

Comment:
FYI, you can validate this with the first molecule job. 
I suggest that the first molecule test should run the task that imports the external <service>-config-ansible-role.